runtime/buffer_test.cc

```diff
@@ -424,4 +424,340 @@ TEST(buffer, copy) {
   test_copy<big>();
 }
 
+TEST(buffer, for_each_contiguous_slice_multi) {
+  buffer<char, 3> buf({10, 20, 30});
+  buffer<char, 3> buf2({10, 20, 30});
+  buf.allocate();
+  buf2.allocate();
+  int slices = 0;
+  for_each_contiguous_slice(
+      buf,
+      [&](void* slice, index_t slice_extent, void* slice2) {
+        memset(slice, 7, slice_extent);
```

Comment:
A copy would be more interesting, and I think a more thorough test too.

Reply:
Done